deeprank2/neuralnets/cnn/model3d.py

```diff
@@ -23,7 +23,16 @@
 # ----------------------------------------------------------------------
 
 
-class CnnRegression(nn.Module):  # noqa: D101
+class CnnRegression(nn.Module):
+    """Convolutional Neural Network architecture for regression.
+
+    This type of network is used to predict a single scalar value of a continuous variable.
```

Comment:
I think generally speaking, it would be nice if all networks had a one-line explanation like this. This is not the case for most of the GNNs.